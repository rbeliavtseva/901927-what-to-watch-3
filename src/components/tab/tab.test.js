import React from "react";
import renderer from "react-test-renderer";
import Tab from "../tab/tab.jsx";

const label = `Show more`;

it(`Render Tab`, () => {
  const tree = renderer
    .create(
        <Tab
          onClick={() => {}}
          activeTab={label}
          label={label}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
