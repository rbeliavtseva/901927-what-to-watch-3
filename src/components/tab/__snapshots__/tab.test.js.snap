// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Tab 1`] = `
<li
  className="movie-nav__item movie-nav__item--active"
  onClick={[Function]}
>
  <a
    className="movie-nav__link"
    href="#"
  >
    Show more
  </a>
</li>
`;
